File: lab3/lab3_Java/src/Task1.java
import java.util.Scanner;

public class Task1 {
    public static void task1() {
        Scanner scanner = new Scanner(System.in);
        double x, u, s = 1, eps = 0.00001;
        System.out.println("Введіть значення x");
        x = scanner.nextDouble();
        u = x/4; int n = 2;
        while(Math.abs(u)>eps) {
            s += u; u *= -x*(4*n-5)/(4*n);
            n++;
        }
        System.out.printf("Значення суми s = %.6f\n", s);
        System.out.printf("Значення функції: %.6f\n",
                Math.pow(1+x,0.25));
        System.out.println("Кількість доданків " + (n - 1));

    }
}

File: lab3/lab3_Kotlin/src/task1.kt
import java.util.Scanner

fun task1() {
    val scanner = Scanner(System.`in`)
    var x: Double
    var u: Double
    var s = 1.0
    val eps = 0.00001
    println("Введіть значення x")
    x = scanner.nextDouble()
    u = x / 4
    var n = 2
    while (Math.abs(u) > eps) {
        s += u
        u *= -x * (4 * n - 5).toDouble() / (4 * n)
        n++
    }
    println("Значення суми s = %.6f".format(s))
    println("Значення функції: %.6f".format(Math.pow(1 + x, 0.25)))
    println("Кількість доданків " + (n - 1))
}
